<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatabaseWindow</class>
 <widget class="QMainWindow" name="DatabaseWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>640</width>
      <height>35</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>640</width>
      <height>35</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>640</width>
      <height>35</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:black;color:white;</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </widget>
   <widget class="QWidget" name="body" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>34</y>
      <width>640</width>
      <height>445</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:black;color:white</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QWidget" name="vw_1" native="true">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>445</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>445</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QLabel" name="photoLabel_1">
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>100</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>160</width>
            <height>100</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="textLabel_1">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="photoLabel_2">
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>100</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>160</width>
            <height>100</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="textLabel_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="photoLabel_3">
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>100</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>160</width>
            <height>100</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="textLabel_3">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="vw_2" native="true">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>445</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>445</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="vw2">
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QLabel" name="photoLabel_4">
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>100</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>160</width>
            <height>100</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="textLabel_4">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="photoLabel_5">
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>100</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>160</width>
            <height>100</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="textLabel_5">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="photoLabel_6">
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>100</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>160</width>
            <height>100</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="textLabel_6">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="vw_3" native="true">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>445</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>445</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="vw3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>40</number>
        </property>
        <item>
         <widget class="QPushButton" name="preButton">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>120</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>120</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>30</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color:white;background-color:#FF8C00</string>
          </property>
          <property name="text">
           <string>上一页</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="nextButton">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>120</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>120</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>30</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color:white;background-color:#FF8C00</string>
          </property>
          <property name="text">
           <string>下一页</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="retButton">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>120</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>120</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>30</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color:white;background-color:#FF8C00</string>
          </property>
          <property name="text">
           <string>返回</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
